queries/queries.go

```diff
@@ -79,6 +80,16 @@ type DistributedQuery struct {
 	Expiration    time.Time
 }
 
+// NodeQuery links a node to a query
+type NodeQuery struct {
```

Comment:
Take a look at other tables defined by gorm, you can add here the `gorm.Model` which will automatically add to the table the following columns:
```
    ID uint `gorm:"primarykey"`
    CreatedAt time.Time
    UpdatedAt time.Time
    DeletedAt DeletedAt `gorm:"index"`
```